### ISSUE TITLE
Feature Request-Crop UI enhancements

### ISSUE THREAD

**Issue by fjdemos** [cyanfish/naps2#34]
Please allow manual perspective correction in the crop UI interface so we can mark 4 corners (not necessarily in a perfect rectangular shape) and the image should be straightened. For reference, mobile apps such as genius scan and CamScanner let us do that.

**Comment by kintaro1981** [comment]
yes! this can be really a game changer as there are no desktop apps (foss) that do it but only mobile apps.
like this: https://github.com/Faiziev/FixPerspective

Demo:
https://fixperspective.vercel.app/

**Comment by bbk0416** [comment]
@kintaro1981 — you mentioned in April that desktop perspective-correction options are still missing and that mobile apps are often the workaround.

I'm checking a narrower workflow and don't want to assume it's the same problem: do you personally use NAPS2 on Windows to import existing scans/images and then still have to do cleanup such as crop/perspective correction/resize outside NAPS2 before saving or sharing?

If yes, which one step is still the repeated manual part today? If not, that's useful too — I'm trying not to broaden this issue into a different workflow.
